### ISSUE TITLE
Shifted columns vs. header

### ISSUE THREAD

**Issue by katush** [lineupjs/lineup_app#30]
If the windows size is smaller than the table width and the horizontal scrolling is necessary, the columns in the part that is accessible only by scrolling will be shifted.
Steps: 
Downsize your window (in my case the window size was 1041*832)
Load taggle demo (https://taggle.caleydoapp.org/#soccer)
Scroll to access the matrix columns

![shifted header](https://user-images.githubusercontent.com/8986080/45353222-9f8a8500-b5ba-11e8-9f32-5c85ef04bac4.gif)

Note: I think I was already reporting this issue once, but I can't find it anymore. I am sorry if this is a duplicate.

**Comment by mstreit** [comment]
@sgratzl, is this still a problem in the latest develop?

**Comment by thinkh** [comment]
In my tests the problem still occurs at https://taggle.caleydoapp.org/#soccer.

**TLDR;**
For performance reasons only DOM elements of the visible cells (= columns) are created. There is a bug in the calculation of the offset when skipping the DOM elements of hidden cells.

**Long version**

I investigated the problem and made the following observations:

* All headers of the ranking are build in advance, even though they might be hidden (by the side panel or by the browser viewport)
  ![image](https://user-images.githubusercontent.com/5851088/48051244-df2db180-e1a4-11e8-9a3f-b843c26d73f5.png)

* The cells of each row are created while scrolling. That's why initially there are only 10 cells available for the row, but 13 for the header.
  ![image](https://user-images.githubusercontent.com/5851088/48051390-42b7df00-e1a5-11e8-806d-393e2a47c0c3.png)

* After scrolling to the right only 9(!) cells of the row are available. The first few are removed
  ![image](https://user-images.githubusercontent.com/5851088/48051545-b528bf00-e1a5-11e8-9b20-9690bae252f4.png)

* Scrolling now back to the left adds all 13 cells, which results in the correct width
  ![image](https://user-images.githubusercontent.com/5851088/48051643-fae58780-e1a5-11e8-8c98-2a0fb2189029.png)

* After that step I'm able to scroll side-ways and always having a correctly aligned header and body

**Comment by mstreit** [comment]
Seems to be solved in the demo version deployed on https://lineup.js.org/app/

**Comment by thinkh** [comment]
I would like to know how @sgratzl solved this problem?
